data/pages-content/cfp.md

```diff
@@ -0,0 +1,164 @@
+---
+title: CFP
+subtitle: EuroPython Call for Proposals
+---
+# Call for Proposals
+
+<div style={{textAlign: "center",  marginBottom: 10}}>
+**EuroPython 2024 Call for Proposals (CFP) is Open.**
+
+It will be running between
+**Wednesday, 14th February 2024** and **Wednesday, 6th March 2024**
+[Anywhere on Earth](https://en.wikipedia.org/wiki/Anywhere_on_Earth)
+</div>
+
+---
+
+EuroPython reflects the colourful and diverse backgrounds, cultures and
+interests of our community, so you (yes, you!) should go for it: **propose
+something and represent!**
+
+No matter your level of Python or public speaking experience, EuroPython's job
+is to help you bring yourself to our community so we all flourish and benefit
+from each other's experience and contribution.
+
+If you’re thinking, “but they don’t mean me”, then we especially mean **YOU**.
+
+* If you’re from a background that isn't usually well-represented in most
+  Python groups, get involved - we want to help you make a difference.
+
+* If you’re from a background which is well-represented in most Python groups,
+  get involved - we want your help making a difference.
+
+* If you’re worried about not being technical enough, get involved - your fresh
```

Comment:
I don't think it's the person that's technical or not. It's their knowledge or the things they want to talk about.
```suggestion
* If you’re worried about not having enough technical knowledge, get involved - your fresh
```

Reply:
It's just an informal way of referring to the same idea, but I like your suggestion.